libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/ui/SalesforceReactActivity.java

```diff
@@ -26,39 +26,46 @@
  */
 package com.salesforce.androidsdk.reactnative.ui;
 
-import android.content.IntentFilter;
 import android.os.Bundle;
+import android.util.Log;
 
 import com.facebook.react.ReactActivity;
-import com.salesforce.androidsdk.accounts.UserAccountManager;
+import com.facebook.react.bridge.Callback;
 import com.salesforce.androidsdk.app.SalesforceSDKManager;
+import com.salesforce.androidsdk.reactnative.bridge.ReactBridgeHelper;
+import com.salesforce.androidsdk.rest.ApiVersionStrings;
 import com.salesforce.androidsdk.rest.ClientManager;
-import com.salesforce.androidsdk.rest.ClientManager.LoginOptions;
 import com.salesforce.androidsdk.rest.ClientManager.RestClientCallback;
 import com.salesforce.androidsdk.rest.RestClient;
+import com.salesforce.androidsdk.rest.RestRequest;
+import com.salesforce.androidsdk.rest.RestResponse;
 import com.salesforce.androidsdk.security.PasscodeManager;
 import com.salesforce.androidsdk.util.EventsObservable;
 import com.salesforce.androidsdk.util.EventsObservable.EventType;
-import com.salesforce.androidsdk.util.UserSwitchReceiver;
 
 /**
  * Super class for all Salesforce activities.
  */
 public abstract class SalesforceReactActivity extends ReactActivity {
 
+    private static final String TAG = "SfReactActivity";
+
+    private RestClient client;
+    private ClientManager clientManager;
     private PasscodeManager passcodeManager;
-    private UserSwitchReceiver userSwitchReceiver;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        Log.i(TAG, "onCreate called");
         super.onCreate(savedInstanceState);
 
+        // Get clientManager
+        clientManager = buildClientManager();
+
         // Gets an instance of the passcode manager.
         passcodeManager = SalesforceSDKManager.getInstance().getPasscodeManager();
-        userSwitchReceiver = new ActivityUserSwitchReceiver();
```

Comment:
Gotcha. Maybe we should add a ```TODO:``` somewhere in this class, so we don't forget to add it back.